specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/datafactory.json

```diff
@@ -2783,6 +2883,28 @@
         "factoryName"
       ]
     },
+    "CreateLinkedIntegrationRuntimeRequest": {
+      "description": "The linked integration runtime information.",
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "The name of the linked integration runtime.",
+          "type": "string"
+        },
+        "subscriptionId": {
+          "description": "The subscription ID for which the linked integration runtime belong to.",
```

Comment:
"The ID of the subscription that the linked integration runtime belongs to."

Reply:
Done. Thanks.